api/test/wfl/integration/terra_test.clj

```diff
@@ -23,9 +23,12 @@
 
 (deftest test-terra-submission
   (testing "A workflow is created for the entity"
-    (let [submission-id (terra/create-submission terra-url workspace
-                                                 method-configuration-name method-configuration-namespace
-                                                 entity-type entity-name)
+    (let [submission-id (terra/create-submission terra-url
+                                                 workspace
+                                                 method-configuration-name
+                                                 method-configuration-namespace
+                                                 entity-type
+                                                 entity-name)
```

Comment:
Maybe create-submission should take `[url workspace {:keys [mcn mcns et en] :as submission}]` or sumpin?

Reply:
Good idea. I plan to change this namespace a bit and remove the url parameter too! For next change.